projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -56,22 +58,41 @@ export default function SubscriptionControls( {
 } ) {
 	return (
 		<>
+			<div
+				style={ {
+					margin: 20,
+					padding: 24,
+					background: '#f2f2f2',
+					borderRadius: '4px',
+					display: 'flex',
+					alignItems: 'center',
+					justifyContent: 'space-between',
+				} }
+			>
+				<span style={ { marginRight: 12 } }>
+					{ createInterpolateElement(
+						sprintf( '<span>%s readers</span> are\xA0subscribed', subscriberCount, 'jetpack' ),
+						{ span: <span style={ { textDecoration: 'underline' } } /> }
+					) }
+				</span>
+				<JetpackLogo size="16" className="wp-block-jetpack-subscriptions__logo" />
```

Comment:
We should have a way to render the monochrome variant without having to supply CSS.

Reply:
```
<JetpackLogo logoColor="#000" size="16" className="wp-block-jetpack-subscriptions__logo" />
```

Reply:
It appears there are more than one `JetpackLogo` component with different sets of props ...